src/activities/bargame/Bargame.qml

```diff
@@ -0,0 +1,388 @@
+/* GCompris - bargame.qml
+ *
+ * Copyright (C) 2016 UTKARSH TIWARI <iamutkarshtiwari@kde.org>
+ *
+ * Authors:
+ *   Yves Combe (GTK+ version)
+ *   UTKARSH TIWARI <iamutkarshtiwari@kde.org > (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.1
+import GCompris 1.0
+
+import "../../core"
+import "bargame.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Image {
+        id: rootWindow
+        source: "qrc:/gcompris/src/activities/bargame/resource/school_bg1.svg"
```

Comment:
why don't use you the bar.level here as I said in the previous review? It would avoid having to change the background manually at each level change. You can keep also the Activity.url that would avoid to change every image paths if we need to change it for some reason